src/commonMain/kotlin/exchange.dydx.abacus/output/Account.kt

```diff
@@ -400,6 +406,7 @@ data class SubaccountPosition(
                         existing.resources !== resources ||
                         existing.childSubaccountNumber !== childSubaccountNumber ||
                         existing.freeCollateral !== freeCollateral ||
+                        existing.marginUsage !== marginUsage ||
```

Comment:
i see, i'm curious if this architecture was adopted in response to actual perf problems or if it's more theoretical? it is not a common pattern and sort of fragile with new devs onboarding, i could easily see someone accidentally creating new objects instead of checking against existing.